src/hooks/useVibeMediaQuery/index.js

```diff
@@ -0,0 +1,19 @@
+import { useLayoutEffect, useState } from "react";
+import useMediaQuery from "hooks/useMediaQuery";
+import { MEDIA_QUERY_SIZES, LARGE, SMALL1, XLARGE, MEDIUM1, MEDIUM2, SMALL2 } from "utils/media-query-utils";
+
+export default function useVibeMediaQuery() {
+  const [mediaSize, setMediaSize] = useState(SMALL1);
+  const [isSmall1, isSmall2, isMedium1, isMedium2, isLarge, isXLarge] = useMediaQuery(MEDIA_QUERY_SIZES);
+
+  useLayoutEffect(() => {
+    if (isSmall1) setMediaSize(SMALL1);
```

Comment:
why not use terminology which related to the kind of screen we are using? Ipad ? Phone? Computer screen?

Reply:
we are not targeting specific types - as they change and there is no clear definition for what it is a tables/phone...., more than that - you can change orientation in tables, phones and resize screen so it is not relevant